### ISSUE TITLE
Miner actor can try to acquire randomness from a future epoch

### ISSUE THREAD

**Issue by arajasek** [filecoin-project/specs-actors#406]
This can happen when trying to setup the first proving period in `handleProvingPeriod`, since we do:

```
assignmentSeed := rt.GetRandomness(crypto.DomainSeparationTag_WindowedPoStDeadlineAssignment, deadline.PeriodEnd(), nil)
```

(https://github.com/filecoin-project/specs-actors/blob/master/actors/builtin/miner/miner_actor.go#L1074)

Based on a quick chat with @anorth, it will probably be sufficient to use `min(deadline.PeriodEnd(), currEpoch - <lookback constant>)` when doing this. The lookback constant can probably just be 1.

**Comment by arajasek** [comment]
Just realised that the randomness being drawn into `assignmentSeed` isn't actually used anywhere. Is that incorrect, or can we just simply get rid of the line (and the parameter in `AssignNewSectors`)?

**Comment by anorth** [comment]
Don't get rid of it. The usage of it hasn't been implemented yet: #409

**Comment by arajasek** [comment]
Nice, got it
